test/test_jit.py

```diff
@@ -2224,6 +2224,28 @@ def forward(self, input):
             with self.assertRaisesRegex(RuntimeError, "is a zip"):
                 torch.load(f.name)
 
+    @unittest.skipIf(not PY35, "Python 3.5 needed")
```

Comment:
Why is Python 3.5 needed ? also might be good to assert that the initial PYTORCH_JIT == PYTORCH_JIT after reloading

Reply:
Usually better to feature test than to hard code a specific Python version. Intent is clearer too.

Reply:
The latest commit removes the test, the change is pretty small and the test messing with environment variables / reloading modules is messing up the other tests.